front/components/data_source/DataSourceSyncChip.tsx

```diff
@@ -116,6 +116,16 @@ export default function ConnectorSyncingChip({
             trigger={<Chip color="warning">Synchronization failed</Chip>}
           />
         );
+      case "remote_database_network_error":
```

Comment:
We set this one manually ?

Reply:
For now yes, I think we should also do it from activity but shipping as is so I can handle this one already